src/Components/CreateImageWizardV2/utilities/requestMapper.tsx

```diff
@@ -111,6 +111,7 @@ export const mapRequestToState = (request: BlueprintResponse): wizardState => {
     fileSystem: {
       mode: 'automatic',
       partitions: [],
+      isNextButtonPristine: false,
```

Comment:
Should this be set to `true`? Since we are mapping the request to the wizard state the form has technically not been interacted with as of yet and the Next button should be pristine.